tests/ui/async-await/coroutine-desc.stderr

```diff
@@ -5,6 +5,7 @@ LL |     fun(async {}, async {});
    |     --- --------  ^^^^^^^^ expected `async` block, found a different `async` block
    |     |   |
    |     |   the expected `async` block
+   |     |   expected argument `f2` to be an `async` block because that argument needs to match the type of this parameter
```

Comment:
I think I'll remove the naming of parameters here as that makes more sense in the note below.